function/templates/TriggerAuthentication.yaml

```diff
@@ -6,5 +6,5 @@ metadata:
 {{- ( include "hmcts.labels.v2" . ) | indent 2 }}
 spec:
   podIdentity:
-    provider: {{ .Values.triggerPodIdentityProvider }} # none | azure | gcp | spiffe | aws-eks | aws-kiam
+    provider: {{ .Values.triggerAuth.triggerPodIdentityProvider }} # none | azure | gcp | spiffe | aws-eks | aws-kiam
```

Comment:
this is a breaking change have you checked if it will affect anyone?

I doubt it so probably okay

Reply:
this will only break if they upgrade to the latest chart version.

I will update the README